Android/src/org/droidplanner/android/fragments/control/BaseFlightControlFragment.java

```diff
@@ -35,6 +38,8 @@
     private static final long FOLLOW_LOCATION_UPDATE_FASTEST_INTERVAL = 5000; // ms
     private static final float FOLLOW_LOCATION_UPDATE_MIN_DISPLACEMENT = 0; // m
 
+    private DroidPlannerPrefs dpPrefs = DroidPlannerPrefs.getInstance(null);
```

Comment:
No need for this instance variable. This fragment has access to the `DroidPlannerPrefs` singleton through the `getAppPrefs()` method defined by its parent.